app/views/mortgage_calculator/stamp_duties/_stamp_duty_to_pay_panel.html.erb

```diff
@@ -12,7 +12,7 @@
     <span class="rendered-from-js">{{ stampDuty.cost() | customCurrency:"£":"whole" }}</span>
   </div>
 
-  <p ng-hide="js" class="stamp-duty__results-tax-rate"><%= I18n.t("stamp_duty.results.sentence", percentage: number_to_percentage(@stamp_duty.percentage_tax, precision: 1)) %></p>
+  <p ng-hide="js" class="stamp-duty__results-tax-rate ng-hide"><%= I18n.t("stamp_duty.results.sentence", percentage: number_to_percentage(@stamp_duty.percentage_tax, precision: 2)) %></p>
```

Comment:
Should a helper/presenter method be used here, `number_to_percentage(@stamp_duty.percentage_tax, precision: 2)`?

Reply:
I am going to leave this as is for now. number_to_percentage is already being used, just being passed as an argument to the translation.